lib/teiserver/tcp/spring/spring_tcp_server.ex

```diff
@@ -41,6 +41,9 @@ defmodule Teiserver.SpringTcpServer do
   def start_link(opts) do
     mode = if opts[:ssl], do: :ranch_ssl, else: :ranch_tcp
 
+    # https://www.erlang.org/docs/23/man/erlang#process_flag_max_heap_size
+    max_heap_size = 2_620_000
```

Comment:
maybe a comment as to what is this number, or replace it with an expression like `1024 * 1024 * 20`